src/api/app/jobs/project_create_auto_cleanup_requests.rb

```diff
@@ -11,7 +11,7 @@ def perform
     return unless cleanup_days && cleanup_days > 0
 
     # defaults
-    User.session = User.current || User.find_by_login('Admin')
+    User.session = User.find_by!(login: 'Admin') unless User.session
```

Comment:
[nitpick]
```ruby
User.session ||= User.find_by!(login: 'Admin')
```
:bowtie:

Reply:
I think this even was the original code more or less - but as I first had User.session= and only afterwards User.session, this went through phases. There are more such places where ||= is easier to read